plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/data/DBDValueHandler.java

```diff
@@ -117,4 +119,13 @@ Object createNewValueObject(@NotNull DBCSession session, @NotNull DBSTypedObject
     @NotNull
     DBCLogicalOperator[] getSupportedOperators(@NotNull DBSTypedObject attribute);
 
+    /**
+     * Get comparator for compare values
+     *
+     * @return comparator
+     */
+    default Comparator<Object> getComparator() {
+        return null;
+    }

```

Comment:
Let's not extend the basic interface with such a crude functionality (I don't like it's constrained to `Object`). Instead, let the implementations implement `Comparable` as well.